content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -219,57 +234,63 @@ To configure the Tracer using a JSON file, create `datadog.json` in the instrume
 
 {{< /tabs >}}
 
-### Configuration Variables
+## Configuration settings
 
-The following tables list the supported configuration variables. Use the first name (e.g. `DD_TRACE_AGENT_URL`) when setting environment variables or configuration files. The second name, if present (e.g. `AgentUri`), indicates the name the `TracerSettings` propery to use when changing settings in the code.
+Using the methods described above, customize your tracing configuration with these settings. Use the first name (for example, `DD_<INTEGRATION>_ENABLED`) when setting environment variables or configuration files. The second name (for example, `Enabled`), indicates the name for the `IntegrationSettings` property to use when changing settings in the code. Access these properties through the `TracerSettings.Integrations[]` indexer. 
```

Comment:
I would like to keep the details and instructions shorter - can we consolidate this to be less wordy?

Reply:
There are two different types of settings (some are integration-specific, some are not) and they are accessed differently, so they need different instructions. This is why they are in separate tables. I'm happy with making them shorter and simpler, as long as they are correct.

Reply:
Another option could be to split the env var name (`DD_TRACE_AGENT_URL`) and the C# property name (`AgentUri`) into separate sections (env vars vs code), then the instructions could be less wordy.

Reply:
If we want to keep these all in the same table instead of breaking out, here's my suggestion. @lucaspimentel what do you think?

```suggestion
Using the methods described above, customize your tracing configuration with these settings. When adding settings via environment variables or configuration files, use the first name (for example, `DD_TRACE_AGENT_URL`) . When adding settings via code, use the `TracerSettings` property that corresponds to the second name, if present.
```

Reply:
I have gone with keeping them in the same tables, and using the wording that zach suggested most recently here. Thank you!